jobs/infra/playbook-jenkins.yml

```diff
@@ -128,7 +128,7 @@
         - jenkins
         - adhoc
     - name: upgrade charmcraft
-      command: "snap refresh charmcraft --channel latest/stable"
+      command: "snap refresh charmcraft --channel 2.x/stable"
```

Comment:
we can put charmcraft on the runners themselves -- but it doesn't make much difference.  Charmcraft builds are run from a container on the runner, not the runner themselves